commercetools/provider.go

```diff
@@ -86,27 +92,26 @@ func Provider() terraform.ResourceProvider {
 }
 
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
+	clientID := d.Get("client_id").(string)
+	clientSecret := d.Get("client_secret").(string)
 	projectKey := d.Get("project_key").(string)
-
 	scopesRaw := d.Get("scopes").(string)
-	var scopes []string
-	if scopesRaw == "" {
-		scopes = []string{fmt.Sprintf("manage_project:%s", projectKey)}
```

Comment:
The `manage_project` scope is [not recommended to be used](https://docs.commercetools.com/http-api-scopes#manage_projectprojectkey)

<img width="809" alt="image" src="https://user-images.githubusercontent.com/1110551/75192822-52ba5000-5755-11ea-8a06-66848cff972c.png">

Therefore, we should not provide it as a default value. The user must specify the scopes explicitly.